bundler/lib/bundler/templates/newgem/.rubocop.yml.tt

```diff
@@ -0,0 +1,9 @@
+# This is the default RuboCop configuration.
+
```

Comment:
I would remove the above comment since it might have the effect of leading users into thinking that the actual default rubocop configuration is not being applied.

Reply:
Aye, aye. Shall fix.